menus/context-git.json

```diff
@@ -18,6 +18,9 @@
 				"label": "Undo Last Commit",
 				"command": "context-git:undo-last-commit"
 			}, {
+				"label": "Checkout",
+				"command": "context-git:checkout"
+			}, {
```

Comment:
could we change this to `context-git:switch-branch` and change the label to `Switch Branch`